src/components/atoms/GradientBackground/GradientBackground.model.ts

```diff
@@ -3,9 +3,10 @@ export type GradientType = {
   colors: string[];
 };
 
-export interface IProps {
+export interface IProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    React.RefAttributes<HTMLDivElement> {
   background: string;
   children?: JSX.Element[] | JSX.Element | string;
```

Comment:
Hmm.. do we need children props here? I'm not sure but please confirm it. I hope that each react component has possibility to have children props without defining them in interface